data/json/recipes/recipe_others.json

```diff
@@ -886,6 +886,7 @@
     "skill_used": "survival",
     "time": 3000,
     "autolearn": true,
+    "reversable": false,
```

Comment:
```suggestion
     "reversible": false,
```

Reply:
Whoops, if I hadn't typoed it would have thrown an error because reversible = false is redundant, it's already not-reversible.